src/components/Strategies/strategyData.tsx

```diff
@@ -0,0 +1,45 @@
+import { IStrategies } from '../../types';
+export const strategies: IStrategies[] = [
+  {
+    title: 'Bridgeworld (Atlas Mine)',
+    imageSrc: '../img/atlas-mine.jpg',
+    ComingSoon: false,
+    itemid: 1,
+    detailPageURL: 'bridgeworld-claim-and-send',
+  },
+  {
+    title: 'Magic Dragon DAO',
+    imageSrc: './img/magic-dragon.jpg',
+    ComingSoon: true,
+    itemid: 2,
+    detailPageURL: '',
+  },
+  {
+    title: 'BattleFly',
+    imageSrc: './img/battlefly.jpg',
+    ComingSoon: true,
+    itemid: 3,
+    detailPageURL: '',
+  },
+  {
+    title: 'Tales of Elleria',
+    imageSrc: './img/tales-of-alleria.jpg',
+    ComingSoon: true,
+    itemid: 4,
+    detailPageURL: '',
+  },
+  {
+    title: 'SmithyDAO',
+    imageSrc: '../img/smithy.jpg',
+    ComingSoon: true,
```

Comment:
`comingSoon` - why capitalize? You need to be consistent and pay more attention to this